dist/content_script.js

```diff
@@ -416,12 +417,16 @@ scriptElem.text = `
 
   const originalStringify = JSON.stringify;
   JSON.stringify = function(value) {
-    if (value && value.params && value.params.profiles) {
-      value.params.profiles.unshift(WEBVTT_FMT);
-      // console.log('stringify', value);
-    }
-    if (value && value.ab && value.ab.profiles) {
-      value.ab.profiles.unshift(WEBVTT_FMT);
+    if (value && value.url === MANIFEST_URL) {
+      // Try not to hardcode property names here because Netflix 
```

Comment:
Hopefully this update is slightly more robust than the last one, but if this breaks again, we may have to look at something more long term.  I know this is still pretty fragile, but `stringify` is a function that gets called a lot, so I'd hate to add the overhead of searching all of the properties in every request to avoid hard coding property names.